telemetry/util/heka_message_parser.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import ujson as json
```

Comment:
Everything else in telemetry-server uses "simplejson", so we should use that here too, if possible.